.github/workflows/release.yml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - master
```

Comment:
What if we remove master here, and just merge this branch to legacy instead of to master?